powerhome/playbook
# [PBNTR-207] Adding div support for Table kit

**What does this PR do?**
Adding div support for Table kit

**Screenshots:**
![image](https://github.com/powerhome/playbook/assets/2573205/303c29b2-9ad5-4a4d-9520-f48e5f791422)

**How to test?** 
1. Go to the Table Kit page
2. Check the first doc example


#### Checklist:
